dense_vector/challenges/default.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "index-and-search",
+  "description": "indexes vectors, force merges to a single segment, then searches them",
+  "default": true,
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "delete-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": "index-append",
+      "warmup-time-period": {{ bulk_warmup | default(240) | int }},
+      "clients": {{bulk_indexing_clients | default(8)}}
+    },
+    {
+      "operation": {
+        "operation-type": "force-merge",
+        "max-num-segments": 1,
+        "request-timeout": 7200
+      }
+    },
+    {
+      "name": "refresh-after-force-merge",
+      "operation": "refresh"
+    },
+    {
+      "name": "wait-until-merges-finish",
+      "operation": {
+        "operation-type": "index-stats",
+        "index": "_all",
+        "condition": {
+          "path": "_all.total.merges.current",
+          "expected-value": 0
+        },
+        "retry-until-success": true,
+        "include-in-reporting": false
+      }
+    },
+    {
+      "name": "knn-search-10-500",
+      "operation": "knn-search-10-500",
+      "warmup-iterations": 100,
+      "iterations": 100,
```

Comment:
This may be an obvious question, but could you explain why the iterations would scale with ingest percentage? It's not clear to me that there'd be a relationship.

Reply:
> This may be an obvious question, but could you explain why the iterations would scale with ingest percentage? It's not clear to me that there'd be a relationship.

I don't think it's an obvious question, so it's good we are discussing it. As I mentioned earlier it _could_ (but doesn't have to). As you mentioned in [your comment](https://github.com/elastic/rally-tracks/pull/217#issuecomment-989041810):

> I would suggest ingest_percentage: 20, corresponding to 2 million total docs, or roughly 1 million per shard. I will test this locally and update the number of iterations.

So if we update amount of data we ingest, it _might_ make sense to update the number of iterations (and warmup iterations?), and maybe there is a simple formula we could use.
However, I'd caution that evaluating the right amount of warmup and normal iterations depends on the hardware. Since we intend to run these on the [nightly hardware](https://elasticsearch-benchmarks.elastic.co/#), I suggest we make some measurements on similar hardware. Let's sync offline on how to do that.